app/test_with_resources.bzl

```diff
@@ -1,20 +1,21 @@
-def test_with_resources(file):
-  '''
-  Genrule for test files.
+"""
+Genrule for test files.
```

Comment:
I think this is documentation for the method, not the file.

Reply:
Yes, I had shifted to file level as this file do that particular work. 
I should add some file level and revert this.